components/CoverletterInformation.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <address>
+    <div 
+      class="company">
```

Comment:
your line breaks make the code hard to read. It's not easy to tell where your elements start close because you are breaking every attribut, but you are not breaking the closing tag `>`